code/app-util-xdm/build.gradle

```diff
@@ -31,5 +31,5 @@ android {
 
 dependencies {
     implementation project(':edge')
-    implementation 'com.adobe.marketing.mobile:core:2.0.0-SNAPSHOT'
+    implementation "com.adobe.marketing.mobile:core:${rootProject.mavenCoreVersion}"
```

Comment:
Should this be `implementation 'com.adobe.marketing.mobile:core:2.+'` to match the version used in test apps?

Reply:
requires min core 2.0.0 for timeutils, so probably does not need to be on latest version